core/src/main/scala/akka/kafka/ProducerMessage.scala

```diff
@@ -6,12 +6,18 @@
 package akka.kafka
 
 import org.apache.kafka.clients.producer.{ProducerRecord, RecordMetadata}
+import scala.collection.immutable.Seq
```

Comment:
I think it's better to `import scala.collection.immutable` and then use `immutable.Seq`, to avoid accidental loss of import which can easily happen when merging conflicting changes and such